### ISSUE TITLE
About data set

### ISSUE THREAD

**Issue by kosuketa** [A3RT/2018NAIST_A3RT#1]
As far as I can see,  there is no label after about 420 raws in the training set.  
Also I cannot tell what and which are the labels in the test set. 

It would be helpful if you could reply as soon as possible since there is a limited time until the deadline.
